### PR TITLE
jsk_fetch_startup: add voice_text and speak-jp

### PR DESCRIPTION
**Pull request by furushchev**
- Installed voice text to fetch15.
- Added text to speech nodes to bringup launch file
- Moved `:speak` methods to robot interface (require https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/318 )

### CODE REVIEW

File: jsk_fetch_robot/fetcheus/fetch-interface.l [hunk 1]
```diff
@@ -130,9 +130,15 @@
       (setq result (send gripper-action :get-result))
       result))
   ;;
-  (:speak (text &rest args)
-    (let ()
-      (apply #'speak-en text :topic-name "/sound_play" args)))
+  (:speak-en (text &rest args)
```
Comment (k-okada) [inline jsk_fetch_robot/fetcheus/fetch-interface.l]: is this common interface with pr2 robot or robot-interface?
Cc: @snozawa

Comment (furushchev) [reply jsk_fetch_robot/fetcheus/fetch-interface.l]: @k-okada This is not common now but uses the same functions as PR2. AFAIK, this style is first emerged in `naoqieus` https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_naoqi_robot/naoqieus/naoqi-interface.l#L250 .

**Comment by k-okada** [comment]
so, I think we're better to add this method to robot-interface.l

--
◉ Kei Okada


2017-09-07 22:19 GMT+09:00 Furushchev <notifications@github.com>:

**Comment by furushchev** [comment]
@k-okada Thanks for advice! I created PR https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/318 to move these methods and removed `:speak` from fetch-interface instead.

**Pull request closed without merging**